src/translations/it/translations.ts

```diff
@@ -312,6 +312,14 @@ export const TRANSLATIONS_IT = {
       title: "L'operazione è stata presa in carico",
       body: "Riceverai l’esito del pagamento all'indirizzo {{useremail}}",
     },
+    survey: {
```

Comment:
Suggestion: isn't better to _name_ this with a more specific label?
Try to imagine a situation where you've more surveys in the webapp... how manage this in a structured way?

Reply:
I think it's not necessary because survey is a property of paymentResponsePage and we can distinguish several surveys on different pages like the example below
`paymentResponsePage.survey
 paymentCheckPage.survey`